# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: xpcom/tests/gtest/TestThreadMetrics.cpp
# Commit: 36b0646e8649
# Full Hash: 36b0646e86498662e770756ad54b1fd648a82184
# Author: Andreas Farre <farre@mozilla.com>
# Date: 2020-04-07 21:44:02
# Regressor Bug: 1620594
# File Overlap Count: 1
# Description:
#   Bug 1620594 - Part 7: Remove TabGroup and SystemGroup. r=nika,bas
#   
#   TabGroup never really made any difference in which thread something go
#   dispatched to. This was the intended use, but development of TabGroups
#   with abstract main threads never made it that far. The good thing is
# ==============================================================================

diff -r a5ad3b179c92 -r 36b0646e8649 xpcom/tests/gtest/TestThreadMetrics.cpp
--- a/xpcom/tests/gtest/TestThreadMetrics.cpp	Tue Apr 07 15:17:14 2020 +0000
+++ b/xpcom/tests/gtest/TestThreadMetrics.cpp	Tue Apr 07 15:17:47 2020 +0000
@@ -9,7 +9,6 @@
 #include "mozilla/AbstractThread.h"
 #include "mozilla/Preferences.h"
 #include "mozilla/dom/DocGroup.h"
-#include "mozilla/dom/TabGroup.h"
 #include "mozilla/SchedulerGroup.h"
 #include "mozilla/TaskCategory.h"
 #include "mozilla/PerformanceCounter.h"
@@ -118,10 +117,10 @@
 
  protected:
   virtual void SetUp() {
-    // building the TabGroup/DocGroup structure
-    RefPtr<dom::TabGroup> tabGroup = new dom::TabGroup(false);
-    mDocGroup = tabGroup->AddDocument(NS_LITERAL_CSTRING("key"), nullptr);
-    mDocGroup2 = tabGroup->AddDocument(NS_LITERAL_CSTRING("key2"), nullptr);
+    // building the DocGroup structure
+    RefPtr<dom::BrowsingContextGroup> group = new dom::BrowsingContextGroup();
+    mDocGroup = group->AddDocument(NS_LITERAL_CSTRING("key"), nullptr);
+    mDocGroup2 = group->AddDocument(NS_LITERAL_CSTRING("key2"), nullptr);
     mCounter = mDocGroup->GetPerformanceCounter();
     mCounter2 = mDocGroup2->GetPerformanceCounter();
     mThreadMgr = do_GetService("@mozilla.org/thread-manager;1");